gallery.markdown

```diff
@@ -54,3 +54,39 @@ order: 1
   }
   </script>
 </figure>
+<figure itemscope="" itemtype="https://schema.org/Painting">
+  <img src="https://github.com/naomimakkelie-nl/www2.naomimakkelie.nl/assets/62353999/47ccca17-7c11-4788-879b-e90da6921c5e" alt="Battlefield on Valentine’s Day" itemprop="image">
```

Comment:
Instead of using the image that was uploaded to GitHub, save the image to the `/assets/img` directory inside the repositor and use the URL `https://www2.naomimakkelie.nl/assets/img/<filename>` to refer to the image.